library/mobjects.c

```diff
@@ -820,7 +822,16 @@ static int eblob_iterate_existing(struct eblob_backend *b, struct eblob_iterate_
 
 int eblob_iterate(struct eblob_backend *b, struct eblob_iterate_control *ctl)
 {
-	return eblob_iterate_existing(b, ctl);
+	if (b == NULL || ctl == NULL)
```

Comment:
it's a bit out of the scope and I doubt that we always need to check input parameters for nullptr

Reply:
discussed it